titanic.py

```diff
@@ -164,10 +149,8 @@ def run_demo():
     penalty = maximum_energy_delta(bqm)
 
     for k in range(1, len(features) + 1):
-        kbqm = bqm.copy()
-        kbqm.update(dimod.generators.combinations(features, k,
-                                                  strength=penalty))  # Determines the penalty
-
+        kbqm = dimod.generators.combinations(features, k, strength=penalty) # Determines the penalty
```

Comment:
[80 char limit](https://www.python.org/dev/peps/pep-0008/#id19). I personally wouldn't put that in pep8, but here we are.

Reply:
Though it does seem we are not consistent about it even in this file... So local (in)consistency trumps global rules.

Reply:
I wondered about that when I was writing it, but it looked like we were using a larger limit. I know some companies set their own. Google went from 80 to 120 while i was there.

Also what do we think about string quotes?

Reply:
Should I run the whole file trough [Black](https://black.readthedocs.io/en/stable/) (in a separate PR)?

Reply:
Interesting, we should discuss it in the next tools meeting IMO. @randomir